en/developer_guide/manage_assets.rst

```diff
@@ -0,0 +1,245 @@
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        require('load-grunt-tasks')(grunt);
+        grunt.loadTasks('./grunt_tasks');
+        grunt.loadTasks('./vendor/open-orchestra/open-orchestra-cms-bundle/OpenOrchestra/GruntTasks');
+
+        var merge = require('merge');
+        var config = {
+            pkg: grunt.file.readJSON('package.json'),
+            env: process.env
+        };
+        config = merge.recursive(true, config, loadConfig('./grunt_tasks/options/'));
+        config = merge.recursive(true, config, loadConfig('./vendor/open-orchestra/open-orchestra-cms-bundle/OpenOrchestra/GruntTasks/Options/'));
+
+        grunt.initConfig(config);
+    };
+
+    function loadConfig(path) {
+        var glob = require('glob');
+        var object = {};
+        var key;
+
+        glob.sync('*', {cwd: path}).forEach(function(option) {
+            key = option.replace(/\.js$/,'');
+            keys =  key.split('.');
+            if (1 == keys.length) {
+                object[keys[0]] = require(path + option);
+            } else {
+                if (!object[keys[0]]) {
+                    object[keys[0]] = {};
+                }
+                object[keys[0]][keys[1]] = require(path + option);
+            }
+        });
+
+        return object;
+    }
+
+This file will load the configuration from your own project (located in the ``grunt_tasks`` folder) and merge
+them with the configuration given by the ``open-orchestra-cms-bundle``.
+
+Adding an Open Orchestra bundle
+-------------------------------
+
+If you take a closer look at the ``open-orchestra-cms-bundle``, it does not contain the media administration part.
+To manage the media, we have created the ``open-orchestra-media-admin-bundle``.
+
+This bundle will come with it's own javascript and css files that need to be added to the website assets files.
+
+You will need to add two part to the ``Gruntfile.js`` file :
+
+- loading the configuration
+- excecuting the specific tasks
+- merging the configuration to the existing configuration
+- add the file generated by the bundle in the concat configuration
+
+.. code-block:: javascript
+
+    // line 5
+    grunt.loadTasks('./vendor/open-orchestra/open-orchestra-media-admin-bundle/OpenOrchestra/GruntTasks');
+
+    // line 12
+    config = merge.recursive(true, config, loadConfig('./vendor/open-orchestra/open-orchestra-media-admin-bundle/OpenOrchestra/GruntTasks/Options/'));
+
+Modify the main javascript task (``grunt_task/javascript_task.js``):
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+        grunt.registerTask('javascript',
+            [
+                'coffee:discovering',
+                'coffee',
+                'concat:smartadminjs',
+                'concat:libjs',
+                'concat:orchestrajs',
+                'concat:mediajs',
+                'concat:js'
+            ]
+        );
+    };
```

Comment:
Reading that, I think those specifics modifications should be described bundle by bundle in a readme file